### PR TITLE
feat(PPDSC-2048): audio player next prev button

### PR DESCRIPTION
**Pull request by mutebg**
PPDSC-2048

**What**
What did you do:
- added the next and prev button, they are enabled only when the user provide onClick function; or back button is enabled after 5 sec of the audio current time
- updated PlayPause button to fallow same getter implementation as other buttons

**I have done:**
 - [x] Written unit tests against changes
 - [ ] Written accessibility tests against the component
 - [x] Written functional tests against the component and/or NewsKit site
 - [ ] Updated relevant documentation
 - [ ] Updated the release notes for the [next release version](https://nidigitalsolutions.jira.com/wiki/spaces/NPP/pages/2349695021/Release%2BNotes%2BCode%2B-%2BWeb%2BDocumentation)
 - [ ] Discussed with QA and/or design about any visual changes

**I have tested manually:**
 - [ ] Renders correctly on IE11
 - [ ] The feature's functionality is working as expected on other browsers such as Safari, Firefox and IE11
 - [ ] There are no accessibility errors reported by WAVE
 - [ ] The screen reader reads the expected text for the tabbable elements.
 - [ ] There are no new errors in the browser console coming from this PR.
 - [ ] When visual test is not added, it renders correctly on different browsers and mobile viewports (Safari, Firefox, small mobile viewport, tablet)
 - [ ] The Playground feature is working as expected

**Before**

**After:**

**Who should review this PR**

**How to test**

### CODE REVIEW

**Comment by newskit-engineering** [comment]
You can preview these changes on: 
  Documentation Site: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2048-next-prev-button 
  Storybook: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2048-next-prev-button/storybook

File: src/audio-player-composable/audio-player-composable.tsx [hunk 1]
```diff
@@ -111,21 +115,19 @@ export const AudioPlayerComposable = ({
...
+      'aria-label': ariaLabel,
+      'aria-pressed': ariaPressed,
       loading,
       onClick,
+      children: playStateIcon,
+      ...getterProps,
 
       // can  be needed for custom internal logic
       playing,
       canPause: live,
```
Comment (Xin00163) [inline src/audio-player-composable/audio-player-composable.tsx:130]: Should it be `canPause: notLive`?

Comment (Xin00163) [reply src/audio-player-composable/audio-player-composable.tsx:130]: It's not added in your pr but could you please change the play/stop button to play/pause button in the play-pause-button.tsx and also update this part:
```
      if (canPause) {
        playStateIcon = <IconFilledPause />;
        ariaLabel = 'Pause';
      } else {
        playStateIcon = <IconFilledStop />;
        ariaLabel = 'Stop';
      }
```

**Review by baburay23 — COMMENTED** [review]
looks good! happy to approve once design review passes

File: src/audio-player-composable/audio-player-composable.tsx (hunk 1 above)
Comment (Xin00163) [reply src/audio-player-composable/audio-player-composable.tsx]: Looks good now. Happy to approve once the tests are updated

Comment (mutebg) [reply src/audio-player-composable/audio-player-composable.tsx]: @Xin00163 done

**Pull request merged**